plugins/woocommerce-blocks/assets/js/types/type-defs/cart.ts

```diff
@@ -203,6 +203,7 @@ export interface Cart extends Record< string, unknown > {
 }
 export interface CartMeta {
 	updatingCustomerData: boolean;
+	applyingExtensionCartUpdates: number;
```

Comment:
Nitpic: I wonder if we can find a more speaking name. To me, _"applying...."_ indicates a boolean value. As we're checking against a numeric value here, I wonder if we should use _"...count"_, e.g. _"extensionCartUpdatesCount"_ instead?

Reply:
> To me, "applying...." indicates a boolean value

I disagree, `isApplying...` would typically indicate a boolean value, I think `extensionCartUpdatesCount` is fine, I'll update

Reply:
Thank you for improving the prop name, Thomas.